lib/slack.ml

```diff
@@ -222,11 +187,9 @@ let generate_issue_notification ~slack_match_func notification channel =
     sprintf "<%s|[%s]> Issue #%d %s %s by *%s*" repository.url repository.full_name number (pp_link ~url:html_url title)
       action sender.login
   in
-
   make_message ~text:summary ?attachments:(format_attachments ~slack_match_func ~footer:None ~body) ~channel ()
 
-let generate_issue_comment_notification ~(ctx : Context.t) ~slack_match_func ~get_thread_permalink notification channel
-    =
+let generate_issue_comment_notification ~(ctx : Context.t) ~slack_match_func notification channel =
```

Comment:
When testing, PR where threaded fine, but comments on an issue were not put into a thread at all, everything was flat. By reading the code here I do not understand why.

Reply:
looks like issue open did not initialize thread state in #161 , try now

Reply:
Yes, it works as expected now